example-app/src/main/AndroidManifest.xml

```diff
@@ -38,16 +39,21 @@
 
         <activity
             android:name=".ui.card.CardActivity"
-            android:exported="true">
+            android:exported="true"
+            android:launchMode="singleTop">
 
+            <!-- Use a different intent filter then the one used to handle the Drop-In result
+            (if you have both Drop-In and standalone components in your app). If you don't do this
+            the user will have to choose which Activity is used to handle the result, which is a
+            confusing user experience. -->
             <intent-filter>
                 <action android:name="android.intent.action.VIEW" />
 
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
 
                 <data
-                    android:host="${applicationId}"
+                    android:host="${applicationId}.card"
```

Comment:
Can you try to keep the same `android:host` but set `android:path="/card"`? IMO it looks cleaner to use the same host for the same application. The return URL would then be `adyencheckout://com.adyen.checkout.example/card`. WDYT?

Reply:
I like it 👌